src/paperwork/frontend/mainwindow/__init__.py

```diff
@@ -1644,6 +1646,7 @@ def _do_import(self, file_uris):
             lambda _, stats, doc: GLib.idle_add(self.__import_ok, stats)
         )
         self.__main_win.schedulers['main'].schedule(job_importer)
+        self.file_uris = file_uris
```

Comment:
Good point.
It would be cleaner if ```ask_confirmation``` accepted extra arguments as ```**kwargs```, and would then pass them back to ```_ask_confirmation_goto_next``` which would then pass them again to```next_func```.